kubernetes/linux/setup.sh

```diff
@@ -9,88 +9,66 @@ else
     ARCH=$1
 fi
 
-#Download utf-8 encoding capability on the ama-logs container.
-#upgrade apt to latest version
-apt-get update && apt-get install -y apt && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
+sudo tdnf install ca-certificates-microsoft -y
+sudo update-ca-trust
 
+sudo tdnf install ruby-3.1.3 -y
 
-curl -sSL https://rvm.io/mpapis.asc | gpg --import -
-curl -sSL https://rvm.io/pkuczynski.asc | gpg --import -
-curl -sSL https://get.rvm.io | bash -s stable
-
-# setup paths for ruby and rvm
-if [ -f /etc/profile.d/rvm.sh ]; then
-    source /etc/profile.d/rvm.sh
-    echo "[ -f /etc/profile.d/rvm.sh ] && source /etc/profile.d/rvm.sh" >> ~/.bashrc
-fi
-
-rvm install 3.1.3
-rvm --default use 3.1.3
-
-sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
-    dpkg-reconfigure --frontend=noninteractive locales && \
-    update-locale LANG=en_US.UTF-8
-
-#install oneagent - Official bits (3/14/2022)
 if [ "${ARCH}" != "arm64" ]; then
```

Comment:
in linuxrpm.py we are using aarch64, do we want to unify the ARCH name here as well?

Reply:
What do you mean by unify ? The else part is for aarch64

Reply:
I think for making that change we will have to make changes in a lot of other places like the azure pipeline. I will create another PR after this one to address that changes.